dll/win32/msgina/lang/ro-RO.rc

```diff
@@ -220,47 +220,47 @@ STRINGTABLE
+    IDS_PASSWORDMUSTCHANGE "Trebuie să vă schimbați parola la prima Log on."
     IDS_PASSWORDEXPIRED "Parola v-a expirat și trebuie schimbată."
     IDS_ACCOUNTEXPIRED "Contul v-a expirat, adresați-vă administratorului."
     IDS_ACCOUNTLOCKED "Nu vă pot conecta deoarece contul dumneavoastră. a fost blocat. Consultați administratorul de sistem."
-    IDS_INVALIDLOGONHOURS "Contul dumneavoastră are restricții de timp care vă împiedică să vă conectați în acest moment. Încercați mai târziu."
-    IDS_INVALIDWORKSTATION "Contul dumneavoastră este configurat pentru a vă preveni în a folosi acest calculator. Încercați la alt calculator."
-    IDS_ACCOUNTRESTRICTION "Eșec în conectarea dumneavoastră din cauza unei restricții a contului."
-    IDS_EMERGENCYLOGOFFTITLE "Deautentificare din ReactOS"
-    IDS_EMERGENCYLOGOFF "Apăsați Confirmă pentru a vă deautentifica imediat. Toate datele nesalvate vor fi pierdute. Folosiți acest lucru doar ca ultimă soluție."
+    IDS_INVALIDLOGONHOURS "Contul are restricţii de timp care vă împiedică să faceţi Log on acum. Încercaţi din nou mai târziu."
+    IDS_INVALIDWORKSTATION "Contul este configurat să vă împiedice să utilizaţi acest computer. Încercaţi un alt computer."
+    IDS_ACCOUNTRESTRICTION "Nu v-ați putut conecta din cauza unei restricții a contului."
+    IDS_EMERGENCYLOGOFFTITLE "Log off din ReactOS"
+    IDS_EMERGENCYLOGOFF "Apăsați OK pentru a vă deautentifica imediat. Toate datele nesalvate vor fi pierdute. Folosiți acest lucru doar ca ultimă soluție."
     IDS_EMERGENCYRESTARTTITLE "Repornire de urgență"
-    IDS_EMERGENCYRESTART "Apăsați Confirmă pentru a vă reporni calculatorul imediat. Toate datele nesalvate vor fi pierdute. Folosiți acest lucru doar ca ultimă soluție."
+    IDS_EMERGENCYRESTART "Apăsați OK pentru a vă reporni calculatorul imediat. Toate datele nesalvate vor fi pierdute. Folosiți acest lucru doar ca ultimă soluție."
 END
 
 /* Shutdown Dialog Strings */
 STRINGTABLE
 BEGIN
-    IDS_SHUTDOWN_LOGOFF "Deautentificare „%s”"
+    IDS_SHUTDOWN_LOGOFF "Log off din %s"
     IDS_SHUTDOWN_SHUTDOWN "Închidere"
     IDS_SHUTDOWN_RESTART "Repornire"
     IDS_SHUTDOWN_SLEEP "Stare de veghe"
     IDS_SHUTDOWN_HIBERNATE "Hibernare"
     /* Shut down descriptions */
-    IDS_SHUTDOWN_LOGOFF_DESC "Închide sesiunea curentă permițând altor utilizatori autentificarea în sistem."
-    IDS_SHUTDOWN_SHUTDOWN_DESC "Închide atât sesiunea curentă cât și sistemul de operare pentru a puta opri în siguranță alimentarea electrică."
-    IDS_SHUTDOWN_RESTART_DESC "Închide sesiunea curentă și repornește sistemul."
-    IDS_SHUTDOWN_SLEEP_DESC "Induce sistemul în starea de veghe."
-    IDS_SHUTDOWN_HIBERNATE_DESC "Salvează sesiunea curentă și oprește calculatorul."
+    IDS_SHUTDOWN_LOGOFF_DESC "Închide sesiunea curentă permițând altor utilizatori Log on în sistem."
+    IDS_SHUTDOWN_SHUTDOWN_DESC "Se încheie sesiunea şi se închide Windows astfel înc\xE2t oprirea alimentării cu curent să se facă în siguranţă."
```

Comment:
I personally thing the old string was superior here, because it didn't need the \x in înc\xE2t. Is that really a word? But that is just an opinion. I am no native speaker.

Reply:
You're right. I forgot to change it. I will correct it later.

Reply:
Fixed!